src/libraries/System.Text.Json/tests/Serialization/ContinuationTests.NullToken.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    partial class ContinuationTests
+    {
+        // From https://github.com/dotnet/runtime/issues/42070
```

Comment:
Please don't include links to issues; they become irrelevant after a bit; tests should be self-describing.

Reply:
I was told to do so

Reply:
I see no harm in keeping the link as a back-pointer. Some folks go as far as incorporating the issue number in the test name (like FooTest_Regression12345), which I agree isn't great, and this is a reasonable compromise.

We do this elsewhere and there is value in keeping it.